webportal/webapp/src/app/main/register/register.component.ts

```diff
@@ -17,14 +17,7 @@ export class RegisterComponent implements OnInit {
     if (this.email.endsWith('dot.gov')) {
       this.cognitoService.login(false); // Route to ADFS login
     } else {
-      window.location.href = this.buildLoginGovUrl(); // Route to Login.gov
+      window.location.href = this.cognitoService.buildLoginGovRedirectUrl();
     }
   }
-
-  buildLoginGovUrl() {
```

Comment:
This got pulled in from master, but the idea is that this should be encapsulated